src/main/java/com/scaleunlimited/flinkcrawler/fetcher/SimpleHttpFetcherBuilder.java

```diff
@@ -10,19 +10,13 @@
 	public static final int DEFAULT_MAX_SIMULTANEOUS_REQUESTS = 1;
 
 	public SimpleHttpFetcherBuilder(UserAgent userAgent) {
-		super(DEFAULT_MAX_SIMULTANEOUS_REQUESTS, userAgent);
+		this(DEFAULT_MAX_SIMULTANEOUS_REQUESTS, userAgent);
```

Comment:
I think we should just get rid of this constructor, as it leads to problems (and thus no `DEFAULT_MAX_SIMULTANEOUS_REQUESTS`).

Reply:
If so, then I guess I'd be passing 1 to the other constructor for both the robots and the site map fetcher builders, but I'm not sure what I'd have CrawlTopologyBuilder pass when it's initializing _pageFetcherBuilder and _urlLengthener in their declarations (perhaps a CrawlTopologyBuilder.DEFAULT_MAX_SIMULTANEOUS_FETCH_REQUESTS of 10?) I note the similarity here to CrawlTopologyBuilder.DEFAULT_MAX_OUTLINKS_PER_PAGE.

I could also avoid constructing them until the constructor (or even CrawlTopologyBuilder.build time, when I'd only do so if necessary), but then I'd need to force users of this class to pass this parameter in (to either the constructors or a setMaxSimultaneousRequests method that was required unless you provide these builders).

Thoughts?

Reply:
Hmm, for both robots (for sure) and site map fetchers (less important) you'd want to have significant numbers of threads, right?

As to the actual number to use, maybe some percentage of the regular URL fetcher, with a min of 1 (e.g. do 1/5 of the regular fetcher count)

Reply:
Duh. OK, done.